Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -122,7 +145,7 @@ sub _releases {
 		if ($releaseTypes{uc($releaseType)}) {
 			push @items, _createItem($name, $releaseType eq 'COMPILATION'
 					? [ { searchTags => [@$searchTags, 'compilation:1'], orderBy => $orderBy } ]
-					: [ { searchTags => [@$searchTags, "compilation:0", "release_type:$releaseType"], orderBy => $orderBy } ]);
+					: [ { searchTags => [@searchTags, "compilation:0", "release_type:$releaseType", "album_id:$albumList{$releaseType}"], orderBy => $orderBy } ]);
```

Comment:
I know, my fault to create two variables of almost identical name/type... but did you change this here on purpose, or by accident?

Reply:
An accident...

Reply:
FWIW: I believe storing a list of album IDs and using that in the request is the way to go. Otherwise we'll never get out of that query hell. That mix of roles and release types isn't easily dealt with differently. I've been trying to implement the "only consider `ARTIST` the main artist if there's no `ALBUMARTIST`" logic. But that's getting complicated, too, as I'd have to run additional queries, too.